docs/get_started.md

```diff
@@ -73,16 +78,17 @@ See [Support Matrix](support_matrix.md) for more information.
    pip install "ai-dynamo[all]"
    ```
 
-```{note}
-To ensure compatibility, use the examples in the release branch or tag that matches the version you installed.
-```
+> [!Important]
```

Comment:
This is a [MyST](https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#syntax-directives) vs. Markdown support thing. In order to get the "note" admonition to appear correctly in the html rendered docs (See https://docs.nvidia.com/dynamo/latest/get_started.html#alternative-setup-manual-installation) they need to use the ```{note} (and warning, error etc.) syntax.

Reply:
@nealvaidya resolve?